app/models/job_invocation.rb

```diff
@@ -36,7 +36,7 @@ class JobInvocation < ActiveRecord::Base
   belongs_to :triggering, :class_name => 'ForemanTasks::Triggering'
   has_one :recurring_logic, :through => :triggering, :class_name => 'ForemanTasks::RecurringLogic'
 
-  scope :with_task, -> { joins('LEFT JOIN foreman_tasks_tasks ON foreman_tasks_tasks.id = job_invocations.task_id') }
+  scope :with_task, -> { joins(:task) }
```

Comment:
it was originally added in https://github.com/theforeman/foreman_remote_execution/commit/a9c05c8c7dfe8d6dfcf91c782d9b9985f80a19fb#diff-d3ff599b762e4ff95c9b2776d2be1079R32 in https://github.com/theforeman/foreman_remote_execution/pull/76, maybe @adamruzicka would remember... could be some issue in rails 3 AR or it just slipped through